VVDocumenter-Xcode/Setting/VVDSettingPanelWindowController.m

```diff
@@ -54,6 +56,11 @@ - (void)windowDidLoad
     self.btnUseSpaces.state = (NSCellStateValue)[[VVDocumenterSetting defaultSetting] useSpaces];
 
     self.btnAddSinceToComment.state = (NSCellStateValue)[[VVDocumenterSetting defaultSetting] addSinceToComments];
+    self.mtxSinceOptions.enabled = [[VVDocumenterSetting defaultSetting] addSinceToComments];
+    [self.mtxSinceOptions selectCellAtRow:(NSInteger)[[VVDocumenterSetting defaultSetting] sinceOption] column:0];
+    self.tfSinceVersion.enabled = [[VVDocumenterSetting defaultSetting] addSinceToComments];
+    self.tfSinceVersion.stringValue = [[VVDocumenterSetting defaultSetting] sinceVersion];
```

Comment:
I've also changed the since option store an integer in user defaults, and changed the enum from unsigned to signed. 

https://github.com/onevcat/VVDocumenter-Xcode/pull/169/files#diff-7c21f394d1d038756d75a08fc4d3c47fR11